specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,560 @@
+      ],
+      "discriminator": "projectKind",
+      "properties": {
+        "projectKind": {
+          "type": "string",
+          "description": "The type of the project.",
+          "enum": [
+            "conversation",
+            "workflow"
+          ],
+          "x-ms-enum": {
+            "name": "projectKind",
+            "modelAsString": true
+          }
+        },
+        "topIntent": {
+          "type": "string",
+          "description": "The intent with the highest score."
+        }
+      }
+    },
+    "OrchestratorPrediction": {
+      "type": "object",
+      "description": "This represents the prediction result of an Orchestrator project.",
+      "x-ms-discriminator-value": "workflow",
+      "required": [
+        "intents"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/BasePrediction"
+        }
+      ],
+      "properties": {
+        "intents": {
+          "description": "A dictionary that contains all intents. A key is an intent name and a value is its confidence score and target type. The top intent's value also contains the actual response from the target project.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/TargetIntentResult"
+          }
+        }
+      }
+    },
+    "TargetIntentResult": {
+      "type": "object",
+      "description": "This is the base class of an intent prediction",
+      "discriminator": "targetKind",
+      "required": [
+        "confidenceScore",
+        "targetKind"
+      ],
+      "properties": {
+        "targetKind": {
+          "type": "string",
+          "description": "This discriminator property specifies the type of the target project that returns the response.",
+          "enum": [
+            "luis",
+            "conversation",
+            "question_answering",
+            "non_linked"
+          ],
+          "x-ms-enum": {
+            "name": "targetKind",
+            "modelAsString": true
+          }
+        },
+        "apiVersion": {
+          "type": "string",
+          "description": "The API version used to call a target service."
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "float",
+          "x-ms-client-name": "confidence",
+          "description": "The prediction score and it ranges from 0.0 to 1.0.",
+          "minimum": 0,
+          "maximum": 1
+        }
+      }
+    },
+    "ConversationTargetIntentResult": {
+      "type": "object",
+      "description": "A wrap up of Conversation project response.",
+      "x-ms-discriminator-value": "conversation",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TargetIntentResult"
+        }
+      ],
+      "properties": {
+        "result": {
+          "type": "object",
+          "$ref": "#/definitions/ConversationResult",
+          "description": "The actual response from a Conversation project.",
+          "x-nullable": true
```

Comment:
Avoid the use of `x-nullable: true`. Properties with no value should simply be omitted from the payload.